risc0/zkvm/src/host/api/convert.rs

```diff
@@ -209,6 +210,7 @@ impl From<ProverOpts> for pb::api::ProverOpts {
         Self {
             hashfn: opts.hashfn,
             prove_guest_errors: opts.prove_guest_errors,
+            receipt_format: opts.receipt_format as i32,
```

Comment:
```suggestion
            receipt_format: opts.receipt_format as u32,
```

Reply:
So this one is weird - i tried u32 at first but the compiler complained. It seems like protobuf enums are expressed as signed integers...